apps/api-extractor-model/src/model/ModelReferenceResolver.ts

```diff
@@ -108,7 +109,31 @@ export class ModelReferenceResolver {
         return result;
       }
       if (foundMembers.length > 1) {
-        // TODO: Support TSDoc selectors
+        if (memberReference.selector && memberReference.selector.selectorKind === SelectorKind.Index) {
+          const selectedMembers: ApiItem[] = [];
+
+          const selectorOverloadIndex: number = parseInt(memberReference.selector.selector);
+          for (const foundMember of foundMembers) {
+            if (ApiParameterListMixin.isBaseClassOf(foundMember)) {
+              if (foundMember.overloadIndex === selectorOverloadIndex) {
+                selectedMembers.push(foundMember);
+              }
+            }
+          }
+
+          if (selectedMembers.length === 0) {
+            result.errorMessage = `An overload for ${JSON.stringify(identifier)} was not found that matches`
+              + ` the TSDoc selector ":${selectorOverloadIndex}"`;
+            return result;
+          }
+
+          if (selectedMembers.length === 1) {
+            result.resolvedApiItem = selectedMembers[0];
+            return result;
+          }
```

Comment:
Unsure if it's possible, but can there be more than 1 returned here? Is that an existing case? Or are we just leaning on the catch-all message below?

Reply:
Insightful question! 😉  The short answer is there should never be more than 1.

But my PR doesn't consider overloadable members whose name is an empty string.  For example:

```ts
export interface IExample {
  // call signature #1
  (x: number): number;
  // call signature #2
  (x: number, y: number): number;

  // indexer #1
  [x: number]: string;
  // indexer #2
  [x: string]: boolean | string;
}
```

So theoretically the `:1` selector could get confused between the call signature and the indexer.  But it's not a problem in this PR because I didn't implement those selectors, since they've changed in the new TSDoc syntax.